src/Target.php

```diff
@@ -87,35 +86,36 @@ abstract class Target extends Component
      *
      * The signature of the callable should be `function ($message)`.
      */
-    public $prefix;
+    protected $prefix;
+
     /**
      * @var int how many messages should be accumulated before they are exported.
      * Defaults to 1000. Note that messages will always be exported when the application terminates.
      * Set this property to be 0 if you don't want to export messages until the application terminates.
      */
-    public $exportInterval = 1000;
+    protected $exportInterval = 1000;
     /**
      * @var array the messages that are retrieved from the logger so far by this log target.
      * Please refer to [[Logger::messages]] for the details about the message structure.
      */
-    public $messages = [];
+    protected $messages = [];
     /**
      * @var bool whether to log time with microseconds.
      * Defaults to false.
      */
-    public $microtime = false;
+    protected $microtime = false;
 
     /**
      * @var bool
      */
-    private $_enabled = true;
+    protected $enabled = true;
```

Comment:
Why protected? Shouldn't this be private?

Same for other properties.

Reply:
Done. I'm still struggling a bit with old Yii2 behaviors..

The idea was also to allow changing properties from sub classes by just redeclaring them.
It needs to be done in constructor now, like https://github.com/yiisoft/log/pull/24/files#diff-82e69c7367a8ad62e2b11885c2db3013L180